library/Ivoz/Provider/Domain/Service/CallCsvReport/CreateByScheduler.php

```diff
@@ -45,11 +43,12 @@ public function execute(CallCsvSchedulerInterface $scheduler)
         } catch (\Exception $e) {
             $name = $scheduler->getName();
             $this->logger->error(
-                "Call CSV scheduler #${$name} has failed: "
+                "Call CSV scheduler #${name} has failed: "
                 . $e->getMessage()
             );
 
             if ($report && $report->getId()) {
+                // Does this make sense?
```

Comment:
I image this service wants to update the last execution time no matter if it success or fails. This way the scheduler won't stop on first failure (correct me if I'm wrong but next schedule is calculated from last execution).

Does _finally_ works even if you throw the exception in _catch_?

Don't forget to remove that comment :stuck_out_tongue: